src/main/java/md/miller1995/Dealership/services/CarService.java

```diff
@@ -7,4 +7,5 @@
 public interface CarService {
 
     List<Car> findAllCars();
```

Comment:
For now, we can have a method that returns all the cars in the database, but keep in mind that we don't do that in production. What if our `cars` table has 2 million entries? Our API will get frozen.